frontend/src/Society/Society.module.css

```diff
@@ -0,0 +1,29 @@
+.container {
+    height: 174px;
+    width: 200px;
+    border-radius: 16px;
+    box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.16), 0px 3px 6px rgba(0, 0, 0, 0.23);
+    display: flex;
+    flex-direction: column;
+}
+
+.picture {
+    max-width: 100%;
+    border-radius: 16px 16px 0px 0px;
+    height: 106px;
+    background-size: cover;
+}
+
+.socNameDiv {
```

Comment:
i would recommend against abbreviating class names for the sake of clarity.
`society-name-container` is probably a bit better imo